src/backend/distributed/commands/database.c

```diff
@@ -180,15 +180,32 @@ PreprocessAlterDatabaseStmt(Node *node, const char *queryString,
 
 	AlterDatabaseStmt *stmt = castNode(AlterDatabaseStmt, node);
 
-	EnsureCoordinator();
```

Comment:
Why did you remove this? Do `ALTER DATABASE` statements work from workers now?

Reply:
Added again